index.bs

```diff
@@ -4389,7 +4407,7 @@ enum MLPaddingMode {
 
 dictionary MLPadOptions {
   MLPaddingMode mode = "constant";
-  float value = 0;
+  MLNumber value = 0;
```

Comment:
Since MLNumber includes `unrestricted double`, this value may now be non-finite. Do we need to update the `pad()` function's algorithm to check for this?

Reply:
I think in one of the bugs @fdwr suggested that we use MLNumber where we support non-finite and just `(bigint or double)` where we only support finite values.

Some options:

* `MLNumber` typedef for the common case  and explicit union `(bigint or double)` otherwise 
* No typedefs - explicit unions `(bigint or double)` or `(bigint or unrestricted double)` as needed
* Multiple typedefs - e.g. 
  * `typedef (bigint or unrestricted double) MLNumber`
  * `typedef (bigint or double) MLFiniteNumber`
* Use `MLNumber` everywhere, with further restrictions in prose.

I'm slightly partial to the last option, because I anticipate we'll need prose for NaN everywhere anyway, so also restricting infinities is just a few more words. But expressing restrictions in IDL is also nice, so multiple typedefs would be my second choice.

Reply:
> typedef (bigint or double) MLFiniteNumber

This also sounds good.

Reply:
> Multiple typedefs - e.g.
typedef (bigint or unrestricted double) MLNumber
typedef (bigint or double) MLFiniteNumber

I find that the most clear.

Reply:
Added MLFiniteNumber in a3ff823ab44ef44b128234c873a475134c0c5bf4